api/auth_middleware.py

```
@@ -0,0 +1,133 @@
+    if not authorization or not authorization.startswith("Bearer "):
+        return None
+    
+    try:
+        token = authorization.split("Bearer ")[1]
+        decoded_token = auth.verify_id_token(token)
+        return {
+            "uid": decoded_token.get("uid"),
+            "email": decoded_token.get("email"),
+            "firebase_claims": decoded_token
+        }
+    except Exception:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)